Tabloid/Controllers/PostController.cs

```diff
@@ -20,7 +20,11 @@ public IActionResult Get()
         {
             return Ok(_postRepository.GetAll());
         }
-
+        
+        [HttpGet("{firebaseUserId}")]
```

Comment:
This method should be auth protected.

Reply:
you should get the fbUserId here, out of the token, rather than passing the fbUserId in the url.

Reply:
Took care of it. Thanks!